CI/setup-windows-sdk.sh

```diff
@@ -186,32 +186,21 @@ if [ "$(grep -c "/.luarocks-${MSYSTEM}" ${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/
   echo "  Tweaking location for constructed Luarocks so 32 and 64 bits ones do"
   echo "  not end up in the same place when --tree \"user\" is used..."
   echo ""
-  
+
   cp "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.1.lua" "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.1.lua.orig"
   cp "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.4.lua" "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.4.lua.orig"
   /usr/bin/sed "s|.. \"/.luarocks\"|.. \"/.luarocks-${MSYSTEM}\"|" "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.1.lua.orig" > "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.1.lua"
   /usr/bin/sed "s|.. \"/.luarocks\"|.. \"/.luarocks-${MSYSTEM}\"|" "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.4.lua.orig" > "${MINGW_INTERNAL_BASE_DIR}/etc/luarocks/config-5.4.lua"
-  echo "    Completed" 
+  echo "    Completed"
 else
   echo "  Things have already been setup for Luarocks so 32 and 64 bits ones"
   echo "  do not end up in the same place"
 fi
-echo ""
-echo "  When using lua modules from luarocks and you wish to use a per-user one"
-echo "rather than sharing them with other users on this PC it is recommended that"
-echo "you avoid using the --local option to luarocks but instead use"
-echo "--tree \"user\" {the word user, NOT your username). This is so that if you"
-echo "are building things for other PC systems and are working with more than"
-echo "one 'bitness' or are investigating the different Mingw-w64 environments"
-echo "the rocks compiled for each variant do not end up in the same set of"
-echo "directories - which will break things!"
-echo ""
-echo "You will probably have to tweak the LUA_PATH and LUA_CPATH environmental"
-echo "variables before running Mudlet so that it can find the per-user modules"
-echo "You will likely want to review the output from 'luarocks path --help'"
-echo "for more information (remembering to include the '--lua-version 5.1'"
-echo "command line argument to get the right details when using it!)"
-echo ""
+echo "For per-user Lua modules from LuaRocks:"
+echo "- Use '--tree \"user\"' instead of '--local'"
+echo "- This separates modules for different build environments"
+echo "- Adjust LUA_PATH and LUA_CPATH to find per-user modules"
+echo "- See 'luarocks path --help --lua-version 5.1' for details"
```

Comment:
It seems the `--lua-version 5.1` is not required for this action:
```suggestion
echo "- See 'luarocks path --help' for details"
```